README.md

```diff
@@ -14,9 +14,11 @@ and [again](https://www.practicepython.org/blog/2016/12/20/instagram-filters-pyt
 And [again](https://code.tutsplus.com/tutorials/create-instagram-filters-with-php--net-24504)
 and [again](https://picturepan2.github.io/instagram.css/).
 The problem with the attempts is they mostly deal with manually correcting
-colors.
+colors. For me, it was more interesting to find the solution using
+more robust method and math.
 
-This is the only accurate reconstruction of color filters using a robust method.
+This looks like the only attempt to the accurate reconstruction
+of color filters.
```

Comment:
This looks like the only attempt to provide an accurate color filter reconstruction.